src/main.c

```diff
@@ -2291,6 +2295,19 @@ void on_key(GLFWwindow *window, int key, int scancode, int action, int mods) {
             g->observe2 = (g->observe2 + 1) % g->player_count;
         }
     }
+
+    if (key == GLFW_KEY_COMMA) //Lower Height
+    {
+        if (g->p_height > 2)        
+            g->p_height -= 1;            
+    }
+
+    if (key == GLFW_KEY_PERIOD) //Raise Height
+    {
+        State *s = &g->players->state;
+        s->y += 1;
```

Comment:
You do not seem to reset this y height when lowering. Is this intended?

Reply:
With a height lower than 2, you will get stuck on the blocks and won't be able to walk